NewForce-Cohort-4/tabloid-full-stack-team-magenta
# Ticket #6 Create a post

# Description
As an author, I would like to be able to create Posts so I can share my thoughts with the world. Once a user has logged in, they can now select a "new posts" button in the post screen. This opens a new page to add details about a new post. After required info is entered, a user can save their post and will be taken to the newly created details page for that post.
​
# Type of change

#  Testing Instructions 

Pull down and checkout branch vo-posts
Start both server side in VS and and client side from the client folder
use foo@bar.com and password: ******** to log in to the application and see the home page
Once logged in navigate to the Posts tab in the menubar and click the link
On the post page, click the new "new post" button at the top
That should open a new page with form fields to fill out for a new post
Enter some information, so of the fields are required
Once enough information is entered, click the button to save
If successfully saved, the page will redirect to a link for the details of that new post (details page is currently empty, see Ticket #5)
If you added a a publication date, then the post will appear in the list of published posts under the post tab. If no published date was chosen, then the post should appear in the database but will not be shown to the user right now.

​
# Checklist:
​
